tests/test_service.py

```diff
@@ -130,6 +131,10 @@ def test_mixin_typed_service(mosec_service, http_client):
     assert resp.headers["content-type"] == "application/msgpack"
     assert msgpack.unpackb(resp.content) == 11
 
+    # sleep long enough to make sure all the processes have been checked
+    # ref to https://github.com/mosecorg/mosec/pull/379#issuecomment-1578304988
+    time.sleep(GUARD_CHECK_INTERVAL)
```

Comment:
```suggestion
    time.sleep(GUARD_CHECK_INTERVAL + 1)
```

Tried `GUARD_CHECK_INTERVAL` not enough to catch.